content/docs/tutorials/cloudfx/_index.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: Pulumi Cloud Framework (Preview)
+title: Cloud Framework Preview
```

Comment:
```suggestion
title: Cloud Framework (Preview)
```

I think we want to leave the parentheses for now because the library is still in preview. There is probably a better way to show that the library is in preview, but let's leave this as-is for now.